build-scripts/for-win64/build-static-on-ubuntu.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+SRC_DIR=`dirname $0`/../..
+PARALLEL_PRMS="-j$(nproc)"
+DEBIAN_PKG_NAME=wxwidgets3-mingw-w64-static
+VERSION=${1:-3.2.3}
+BUILD_VERSION=${2:-0.go}
+MINGW64_PREFIX=/usr/x86_64-w64-mingw32
+PKG_DIR=`pwd`/build/win64/${DEBIAN_PKG_NAME}_${VERSION}-${BUILD_VERSION}_all
+
+
+mkdir -p build/win64
```

Comment:
```suggestion
mkdir -p build/win64-static
```

And everyewhere later.

Reason: if someone debugs both build types in a local environment, they should not conflict.

Reply:
Ok, I'll change that.

Reply:
@oleg68 In addition to your suggestions above, I also added some changes to get rid of the deprecated warnings when running the workflow file. It's the same type of changes that are already applied to the GO workflow file.